crates/router/src/connector/paypal/transformers.rs

```diff
@@ -262,15 +356,44 @@ impl TryFrom<&PaypalRouterData<&types::PaymentsAuthorizeRouterData>> for PaypalP
                 let amount = OrderAmount {
                     currency_code: item.router_data.request.currency,
                     value: item.amount.to_owned(),
+                    breakdown: AmountBreakdown {
+                        item_total: Amount {
+                            currency_code: item.router_data.request.currency,
+                            value: item.amount.to_owned(),
+                        },
+                    },
                 };
                 let connector_request_reference_id =
                     item.router_data.connector_request_reference_id.clone();
+                let shipping_address = ShippingAddress {
```

Comment:
Use ForeignFrom impl for PaymentsAuthorizeRouterData -> Option<ShippingAddress> and remove code redundancy